<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>201</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>2. Image Smoothing</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>461</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="button2_1">
          <property name="text">
           <string>2.1 Gaussian blur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button2_2">
          <property name="text">
           <string>2.2  Bilateral filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button2_3">
          <property name="text">
           <string>2.3  Median filter</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>201</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>3. Edge Detection</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>461</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QPushButton" name="button3_1">
          <property name="text">
           <string>3.1 Gaussian Blur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button3_2">
          <property name="text">
           <string>3.2  Sobel X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button3_3">
          <property name="text">
           <string>3.3  Sobel Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button3_4">
          <property name="text">
           <string>3.4  Magnitude</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>201</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>4. Transforms</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>461</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QPushButton" name="button4_1">
          <property name="text">
           <string>4.1 Resize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button4_2">
          <property name="text">
           <string>4.2 Translation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button4_3">
          <property name="text">
           <string>4.3  Rotation, Scaling</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button4_4">
          <property name="text">
           <string>4.4  Shearing</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>1. Image Processing</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>461</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="button1_1">
          <property name="text">
           <string>1.1 Load Image File </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button1_2">
          <property name="text">
           <string>1.2 Color Separation  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button1_3">
          <property name="text">
           <string>1.3 Color Transformation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button1_4">
          <property name="text">
           <string>1.4 Blending</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
